modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/osb-site-initializer-customer-portal-test/src/testFunctional/macros/CustomerPortal.macro

```diff
@@ -265,6 +265,27 @@ definition {
 		CustomerPortal.showCPObject(title = "Koroneiki Account");
 	}
 
+	macro createUserForCP {
+		JSONUser.addUser(
+			userEmailAddress = "test.user@liferay.com",
+			userFirstName = "User",
+			userLastName = "2",
+			userScreenName = "Testing");
+
+		JSONUser.setFirstPassword(
+			agreeToTermsAndAnswerReminderQuery = "true",
+			requireReset = "false",
+			userEmailAddress = "test.user@liferay.com");
+
+		Accounts.openAccountsAdmin();
+
+		LexiconEntry.gotoEntry(rowEntry = "Test Account Portal");
```

Comment:
Same one here. Should it be better if the tester had the option to change the account?

Reply:
Done